.cirrus.yml

```diff
@@ -811,9 +811,9 @@ release_task:
 release_test_task:
     name: "Optional Release Test"
     alias: release_test
-    # Release-PRs are never ever marked with [CI:DOCS] or [CI:BUILD]
-    only_if: *not_build
-    # Allow running manually as part of release PR preperation
+    # Release-PRs always include the word release in the title
+    only_if: $CIRRUS_CHANGE_TITLE =~ 'release' || $CIRRUS_CHANGE_TITLE =~ 'Bump'
```

Comment:
Where are Cirrus regexps documented? In particular:
* is it possible to do `=~ 'release|bump`?
* Are they case-sensitive? I Assume and hope that they're not, and if so, I'd prefer lower-case 'bump' for consistency

Reply:
Last I looked, it's basically not documented at all.   Worse, the underlying implementation is Java (or Javascript) with some really insane/wonky defaults.  I think it might be case-insensitive and am pretty sure it's multi-line by default, but I could be remembering wrong.  Really the only thing to do is try it and see what happens :cry:

Reply:
So with the title `[CI:DOCS] Cirrus: testing BUMP keyword` it does match the regex `'.*((release)|(bump)).*'`.  So it's def. case-insensitve and multi-line default :confounded:  - I'm not sure if the outer parenthesis are required, but I can't argue with the result.

Reply:
With the title `[CI:DOCS] Cirrus: Restrict special/limited-use task`, it seems to do "the right thing" as well.  So I think we got it.  Thanks :grin: